examples/simple/main.go

```diff
@@ -66,6 +66,11 @@ func main() {
 	}
 
 	for _, tracker := range s.GetTracks() {
+		t := tracker.Track()
+		tracker.OnEnded(func(err error) {
+			fmt.Printf("Track (ID: %s, Label: %s) ended with error: %v\n",
+				t.ID(), t.Label(), err)
+		})
 		_, err = peerConnection.AddTrack(tracker.Track())
```

Comment:
Could you please change this to `_, err = peerConnection.AddTrack(t)`?

Reply:
Fixed!